src/main/java/seedu/address/logic/commands/ClientViewCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+
+/**
+ * Views a client identified by its displayed index. Displayed on the widget view box.
+ */
+public class ClientViewCommand extends Command {
+
+    /*Client command in the form client view index, where index points to the index of the client at current location
```

Comment:
```suggestion
    /** Client command in the form client view index, where index points to the index of the client at current location
```
To be more inline with the coding standards 
![image](https://user-images.githubusercontent.com/1027621/95684984-a10a5980-0c27-11eb-8e4b-4f244a66c683.png)